MODULE.bazel

```diff
@@ -345,6 +346,56 @@ pip.parse(
 )
 use_repo(pip, "bazel_pip_dev_deps")
 
+# =========================================
+# C++ dependencies
+# =========================================
+
+# Required only by `--extra_toolchains=@local_config_cc//:cc-toolchain-arm64_windows` from .bazelrc
+cc_configure = use_extension("@rules_cc//cc:extensions.bzl", "cc_configure_extension")
+use_repo(cc_configure, "local_config_cc")
+
+http_archive = use_repo_rule("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+SYSROOT_BUILD_FILE = """
+filegroup(
+    name = "sysroot",
+    srcs = glob(["*/**"]),
+    visibility = ["//visibility:public"],
+)
+"""
+
+http_archive(
+    name = "chromium_sysroot_linux_x64",
+    build_file_content = SYSROOT_BUILD_FILE,
+    sha256 = "84656a6df544ecef62169cfe3ab6e41bb4346a62d3ba2a045dc5a0a2ecea94a3",
+    urls = [
+        "https://commondatastorage.googleapis.com/chrome-linux-sysroot/toolchain/2202c161310ffde63729f29d27fe7bb24a0bc540/debian_stretch_amd64_sysroot.tar.xz",
```

Comment:
May want to consider to run something like https://github.com/DavidZbarsky-at/sysroot-min/blob/master/build.sh instead of using the sysroot directly, that removed 80-90% of files and made the sandbox overhead way less painful